atmo/jobs/views.py

```diff
@@ -120,7 +120,7 @@ def detail_spark_job(request, id):
 def download_spark_job(request, id):
     spark_job = SparkJob.objects.get(pk=id)
     response = StreamingHttpResponse(
-        spark_job.notebook_s3_object['Body'].read(),
+        spark_job.notebook_s3_object['Body'].read().decode('utf-8'),
         content_type='application/x-ipynb+json',
```

Comment:
Without decoding the bytestream the streaming response would not be correctly rendered